docs/usage_general.rst.inc

```diff
@@ -404,7 +404,16 @@ Temporary files (client):
     won't be locally cached.
 
 Temporary files (server):
-    None.
+    A non-trivial (potentially 40-50 MB) amount of data will be stored on the
```

Comment:
does it make sense to give a specific amount here?
guess it could vary widely, depending on repo size and what you do?